modules/core/04-channel/keeper/upgrade_test.go

```diff
@@ -252,9 +252,57 @@ func (suite *KeeperTestSuite) TestChanUpgradeTry() {
 				channel.UpgradeSequence = 5
 				path.EndpointB.SetChannel(channel)
 			},
-			// channel sequence - 1 will be returned (upgrade sequence is bumped in init as this is non-crossing hellos case)
+			// channel sequence will be returned so that counterparty inits on completely fresh sequence for both sides
 			types.NewUpgradeError(5, types.ErrInvalidUpgradeSequence),
 		},
+		{
+			// ChainA(Sequence: 0, ics20-2), ChainB(Sequence: 0, ics20-3)
```

Comment:
comment are maybe a little misleading with the ics20 versions, when in the code we're using `mock.Version` and `mock.UpgradeVersion`

I can push a commit to change the comments around a bit!